### ISSUE TITLE
3 issues with 3 TOP Scripts(?) since 7.1

### ISSUE THREAD

**Issue by Zer0toNine** [PunishXIV/Splatoon#197]
1.
P2 i think its the Limitless Synergy script? i couldn't really pinpoint what causes it but since 7.1 i randomly see(like a 50/50 thing if they spawn) big orange cones when the Playstation markers pop up its always 2 of them and they are attached to random players and they are always pointing towards their tether partner. The cones appear to be the Big Orange cones that Tanks would usually tether-bait the mechanic after that in P2. (first i thought it might be the people that get "Stack" after, but its not its just random people.)

2.
also since 7.1 in P3 the Script that does the coloured tethers (red & blue) that point towards the towers seems to be broken for the last set. What happens is that during the last Mechanic Set the red/green tether would also stack between the Stack towers instead of going far besides the Defamation Tower, but sometimes it will still point you to go beside the Defamation tower. i'm not 100% sure but it seemed to be the case whenever i was Defamation First set, which will let me end up with the Red/green tether in the last set. (It could or could not be tied to a certain defamation colour as well. i saw this happening last time i had Blue Defamation at first set. But i'm sure to recall that when i had red defamation it showed the correct tether, again not a 100% sure on that call.)

3.
this tend to happen even before 7.1 already where the P4 BSOD Adjuster script has a chance of not showing anything for the whole phase.

**Comment by Redmoonwow** [comment]
@Zer0toNine 
My third question is, let me ask a question.
* Have you finished P3 after Ion's chant has started?
* Like the OverSumpledWaveCannon, all members need to be set up, have you done that?

**Comment by Zer0toNine** [comment]
@Redmoonwow 

With Ion's Chant you probably mean the enrage cast right? i guess sometimes we've sandbagged into it. I'm not sure wether or not the script didn't work if we've seen the p3 enrage cast (as that's not really a thing i would've thought about)
I'll try to check in my next raid, if the script doesnt work when we see p3 enrage cast or when we don't. and come back with the answer to that hopefully.

With the Second part you mean that at the start of P4 we all stand in our P4 Spread position? Yeah im certain that my group always does that correctly

**Comment by Redmoonwow** [comment]
@Zer0toNine 
In order to support multiple languages, BSOD Adjuster uses enrage cast as a trigger, not the phase transition dialogue. If it does not appear even after watching the enrage cast chant, it would be difficult to fix without ARR data, so I would appreciate it if you could provide it.

**Comment by Zer0toNine** [comment]
@Redmoonwow yeah it could be that its missing if we're not seeing the enrage cast from what i can tell. So there's no way to fix it other than sandbagging into the enrage cast? 
sorry since ARR was broken the whole time in 7.1 when i was clearing i dont really have any replay data of that
